text_generation/causal_lm/cpp/continuous_batching/library/src/scheduler.hpp

```diff
@@ -104,6 +106,7 @@ class Scheduler {
                 m_block_manager.free_sequence(seq_id);
             }
             sequence_group->reset();
+            sequence_group->set_waiting();
```

Comment:
Not sure that I understand, why is it required.
My understanding was that if sequence group is preempted, then it will not be scheduled on current iteration.

Reply:
Discussed offline, so left as it is.